alf/environments/suite_bsuite.py

```diff
@@ -0,0 +1,106 @@
+import alf
+from alf.environments import gym_wrappers, alf_wrappers, alf_gym_wrapper
+from alf.environments.suite_gym import wrap_env
+
+import bsuite
```

Comment:
Since this is a new dependency, it's probably a good idea to add it to `setup.py` or `requirements.txt` as well.

Reply:
Added bsuite to requirements.txt